### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by LionelColaso**

### CODE REVIEW

File: app/views/mods_panel.py [hunk 1]
```diff
@@ -1331,9 +1331,9 @@ def eventFilter(self, source_object: QObject, event: QEvent) -> None:
                                 mod_metadata = (
                                     self.metadata_manager.internal_local_metadata[uuid]
                                 )
-                                if not mod_metadata[
+                                if mod_metadata[
```
Comment (hk21702) [inline app/views/mods_panel.py:1334]: Is there a potential key error for these?

Comment (LionelColaso) [reply app/views/mods_panel.py:1334]: None at the moment
I am only fixing whatever PEP8 issues Pycharm is complaining about at the moment

Comment (LionelColaso) [reply app/views/mods_panel.py:1334]: if you want i will undo it ?

Comment (LionelColaso) [reply app/views/mods_panel.py:1334]: The difference is in the way the first condition is written:
mod_metadata["data_source"] != "expansion" is a more direct way of checking if the value is not equal to "expansion".
not mod_metadata["data_source"] == "expansion" first checks if the value is equal to "expansion", and then negates the result using the not operator.

File: app/controllers/main_window_controller.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
-from loguru import logger
-
 from PySide6.QtCore import QObject, Slot
 from PySide6.QtWidgets import QPushButton
+from loguru import logger
```
Comment (twstagg) [inline app/controllers/main_window_controller.py:3]: I agree that the whitespace should be removed, but import order groups should be sorted alphabetically.

> Imports should be grouped in the following order:
> 
>     Standard library imports.
>     Related third party imports.
>     Local application/library specific imports.

Your grouping choice is correct, but loguru should be imported before PySide6 imports (L comes before P)

File: app/models/dialogue.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import os
+import warnings
```
Comment (twstagg) [inline app/models/dialogue.py:2]: Same as above. Import groups should be sorted alphabetically.

File: app/views/main_window.py [hunk 4]
```diff
@@ -1,7 +1,6 @@
+import os
```
Comment (twstagg) [inline app/views/main_window.py:1]: Same as above. Import groups should be sorted alphabetically.

File: app/views/mods_panel.py (hunk 1 above)
Comment (twstagg) [reply app/views/mods_panel.py:1334]: I agree that this definitely reads better with Lion's change.. It's really a matter of semantics though, and if it helps us be able to read the code better, I am all for it!

File: app/windows/rule_editor_panel.py [hunk 5]
```diff
@@ -19,6 +18,7 @@
     QVBoxLayout,
     QWidget,
 )
+from loguru import logger
```
Comment (twstagg) [inline app/windows/rule_editor_panel.py:21]: Same as above. Import groups should be sorted alphabetically.

**Review by twstagg — CHANGES_REQUESTED** [review]
Some minor nitpicks.

All and all, great work on updating some of those namespace changes! I think that a lot of that is just left over from PySide2 (which is what we initially had) Thank you for taking the time to catch all of these!

File: app/controllers/main_window_controller.py (hunk 2 above)
Comment (LionelColaso) [reply app/controllers/main_window_controller.py:3]: I just Used Optimize Imports feature in PyCharm and it arranged it

File: app/models/dialogue.py (hunk 3 above)
Comment (LionelColaso) [reply app/models/dialogue.py:2]: I just Used Optimize Imports feature in PyCharm and it arranged it

File: app/views/main_window.py (hunk 4 above)
Comment (LionelColaso) [reply app/views/main_window.py:1]: I just Used Optimize Imports feature in PyCharm and it arranged it

File: app/windows/rule_editor_panel.py (hunk 5 above)
Comment (LionelColaso) [reply app/windows/rule_editor_panel.py:21]: I just Used Optimize Imports feature in PyCharm and it arranged it

File: app/models/dialogue.py (hunk 3 above)
Comment (twstagg) [reply app/models/dialogue.py:2]: `import warnings` should be moved to where it was.

File: app/views/main_window.py (hunk 4 above)
Comment (twstagg) [reply app/views/main_window.py:1]: `import os` should be moved back to where it was.

File: app/windows/runner_panel.py [hunk 6]
```diff
@@ -3,19 +3,19 @@
 from re import compile, search
 
 import psutil
```
Comment (twstagg) [inline app/windows/runner_panel.py:5]: `import psutil` should be below loguru alphabetically.

Comment (twstagg) [reply app/windows/runner_panel.py:5]: `from re import ...` should be below PySide6.

File: app/models/dialogue.py (hunk 3 above)
Comment (LionelColaso) [reply app/models/dialogue.py:2]: Currently all "Import" and " from Import" are grouped and sorted

File: app/views/main_window.py (hunk 4 above)
Comment (LionelColaso) [reply app/views/main_window.py:1]: same as above

**Review by twstagg — APPROVED** [review]

**Pull request merged**